applicationinsights.ts

```diff
@@ -53,17 +53,28 @@ export let defaultClient: TelemetryClient;
  * and start the SDK.
  */
 export function setup(instrumentationKey?: string) {
-    if(!defaultClient) {
-        defaultClient = new TelemetryClient(instrumentationKey);
-        _console = new AutoCollectConsole(defaultClient);
-        _exceptions = new AutoCollectExceptions(defaultClient);
-        _performance = new AutoCollectPerformance(defaultClient);
-        _serverRequests = new AutoCollectHttpRequests(defaultClient);
-        _clientRequests = new AutoCollectHttpDependencies(defaultClient);
+    if(defaultClient) {
+        Logging.info("setup was called previously. Overwriting existing defaultClient");
+
+        // The 'if checks' here aren't necessary, but guard against
+        // any potential abnormal setup
+        if (_console) _console.dispose();
```

Comment:
I'm curious. Do we need to dispose and recreate these? Shouldn't they work just fine if left alone (eg. calling this fn twice only accomplishes changing the current ikey more or less)